video/out/hwdec/hwdec_vaapi.c

```diff
@@ -451,9 +496,11 @@ static void try_format_config(struct ra_hwdec *hw, AVVAAPIHWConfig *hwconfig)
         goto err;
     }
 
-    for (int n = 0; fmts &&
-                    fmts[n] != AV_PIX_FMT_NONE; n++)
+    for (int n = 0; fmts && fmts[n] != AV_PIX_FMT_NONE; n++) {
         try_format_pixfmt(hw, fmts[n]);
+        try_format_upload(hw, fmts[n]);
```

Comment:
Why is this necessary? This is already called by try_format_pixfmt.

Reply:
Forgot to remove this while refactoring stuff.